pkg/minikube/problem/problem_test.go

```diff
@@ -17,10 +17,92 @@ limitations under the License.
 package problem
 
 import (
+	"bytes"
 	"fmt"
+	"strings"
 	"testing"
+
+	"k8s.io/minikube/pkg/minikube/out"
 )
 
+type buffFd struct {
+	bytes.Buffer
+	uptr uintptr
+}
+
+func (b buffFd) Fd() uintptr { return b.uptr }
+
+func TestDisplay(t *testing.T) {
+	buffErr := buffFd{}
+	out.SetErrFile(&buffErr)
+	var tests = []struct {
+		description string
+		problem     Problem
+		expected    string
+	}{
+		{
+			problem:     Problem{},
+			description: "empty",
+			expected: `
+* Error: [] <no value>
+* Suggestion:
+`,
+		},
+		{
+			problem:     Problem{URL: "example.com"},
+			description: "url",
+			expected: `
+* Error: [] <no value>
+* Suggestion: 
+* Documentation: example.com
+`,
+		},
+		{
+			problem:     Problem{ID: "example", URL: "example.com", Err: fmt.Errorf("test")},
+			description: "url, id and err",
+			expected: `
+* Error: [example] test
+* Suggestion: 
+* Documentation: example.com
+`,
+		},
+		{
+			problem:     Problem{Issues: []int{0, 1}},
+			description: "with 2 issues",
+			expected: `
+* Error: [] <no value>
```

Comment:
Could these test cases be updated to always have an error associated to them? This way we test real-world behavior, and don't codify the string output of undefined behavior.

Reply:
fixed